classes/Http/Controller/Reviewer/TalksController.php

```diff
@@ -85,7 +83,9 @@ public function viewAction(Request $request)
             return $this->app->redirect($this->url('admin_talks'));
         }
 
-        return $this->render('reviewer/talks/view.twig', ['talk' => $handler->getProfile()]);
```

Comment:
While at it, I figured it would make sense to streamline how we pass data to the templates - this is an example.